### PR TITLE
refactor: Splitting out process_block tests

### PR DESCRIPTION
**Pull request by akhi3030**
process_block contains a ton of feature specific tests.  This PR splits a couple of them into separate files.  

In particular, it splits the tests for the following features into their own files:
- protocol_feature_account_id_in_function_call_permission
- protocol_feature_fix_contract_loading_cost

Other than moving code around; updating includes; and exposing functions as necessary, there are no functional changes in this PR.

### CODE REVIEW

**Comment by akhi3030** [comment]
In a subsequent PR, we can consider factoring out various modules inside `process_blocks.rs` into their own files.  That should further help with tidying the file up.

File: integration-tests/src/tests/client/features.rs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/// Collection of feature specific tests
```
Comment (matklad) [inline integration-tests/src/tests/client/features.rs:1]:
```suggestion
//! Collection of feature specific tests
```

File: integration-tests/src/tests/client/features/account_id_in_function_call_permission.rs [hunk 2]
```diff
@@ -0,0 +1,94 @@
+use std::path::Path;
+use std::sync::Arc;
+
```
Comment (matklad) [inline integration-tests/src/tests/client/features/account_id_in_function_call_permission.rs:3]:
```suggestion
```

as per stylegide

File: integration-tests/src/tests/client/features/fix_contract_loading_cost.rs [hunk 3]
```diff
@@ -0,0 +1,95 @@
+use assert_matches::assert_matches;
+
```
Comment (matklad) [inline integration-tests/src/tests/client/features/fix_contract_loading_cost.rs:2]:
```suggestion
```

File: integration-tests/src/tests/client/features/fix_contract_loading_cost.rs [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+use near_chain::ChainGenesis;
+use near_chain_configs::Genesis;
+use near_client::test_utils::TestEnv;
+use near_primitives::types::{AccountId, BlockHeight};
+use near_primitives::views::FinalExecutionStatus;
+use nearcore::config::GenesisExt;
+
```
Comment (matklad) [inline integration-tests/src/tests/client/features/fix_contract_loading_cost.rs:9]:
```suggestion
```

**Comment by matklad** [comment]
>In a subsequent PR, we can consider factoring out various modules inside process_blocks.rs into their own files. That should further help with tidying the file up.

Uhu. Specifically, everything that mentions `ProtocolFeature::` is probably worth splittiong out, eg

https://github.com/near/nearcore/blob/db084d1cdbc33eb59bb51d852f5346f425462926/integration-tests/src/tests/client/process_blocks.rs#L2550

**Review by matklad — APPROVED** [review]
LGTM with formatting fixed!

**Comment by matklad** [comment]
needs `cargo fmt`

**Pull request merged**